src/Tests/Microsoft.NET.Build.Tests/GivenThatWeWantToBuildAComServerLibrary.cs

```diff
@@ -29,19 +29,18 @@ public void It_copies_the_comhost_to_the_output_directory()
 
             var buildCommand = new BuildCommand(testAsset);
             buildCommand
-                .Execute()
+                .Execute("/p:ProduceReferenceAssembly=false")
```

Comment:
I only included it in specific tests that were checking for files. I considered adding the ref file to the check but then we'd have to remove that whenever ref got moved to obj. I wasn't sure about putting it in the project itself given that would affect other tests using the same project but I'm starting to lean that way given your comment (as it would limit where we need to go to remove these later once ref is gone from the bin directory)